lib/stripe/resources/application_fee.rb

```diff
@@ -9,16 +9,5 @@ class ApplicationFee < APIResource
 
     nested_resource_class_methods :refund,
                                   operations: %i[create retrieve update list]
-
-    # If you don't need access to an updated fee object after the refund, it's
-    # more performant to just call `fee.refunds.create` directly.
-    def refund(params = {}, opts = {})
```

Comment:
@ob-stripe Where was this method deprecated? What's the ideal replacement for it?

Reply:
@Senjai You can use the static method `ApplicationFee.create_refund` method instead, e.g.:

```ruby
fee = ApplicationFee.retrieve("fee_123")
fee_refund = fee.refund
```

becomes:
```ruby
fee_refund = ApplicationFee.create_refund("fee_123")
```

Cf. the [migration guide for v5](https://github.com/stripe/stripe-ruby/wiki/Migration-guide-for-v5#removed-methods).

Reply:
We replaced this with

```ruby
          application_fee.raw_data.refunds.create(amount: refund_amount.subunits)
          application_fee.refresh
```
But mostly I'm concerned commenting about where this method and others were considered deprecated, just so I can follow along

Reply:
This method (as well as a few others that were removed in v5) was never explicitly tagged as deprecated. We try to give advance notice of deprecations, but major versions will sometimes contain breaking changes without advance notice. Such changes are always documented in the changelog/migration guide though.